Makefile

```diff
@@ -34,12 +42,13 @@ endif
 
 .PHONY: image-build
 image-build: check-image-env ## Build NetObserv collection image.
-	$(OCI_BIN) build --build-arg BUILD_VERSION="${BUILD_VERSION}" -t ${IMAGE_REGISTRY}/${MUST_GATHER_IMAGE}:${IMAGE_TAG} .
+	trap 'exit' INT; \
+	$(foreach target,$(MULTIARCH_TARGETS),$(call build_target,$(target)))
```

Comment:
does docker buildx now allow building for one architecture at time and composing the final manifest-list image? The last time I used it, it was only capable of building and pushing together via `docker buildx build --push --platforms linux/arm64,linux/amd64,....`

Reply:
we been using this pattern on all netobserv repo